lib/src/json_schema/validator.dart

```diff
@@ -321,6 +321,15 @@ class Validator {
     }
   }
 
+  void _validateCustomSetAttributes(JsonSchema schema, Instance instance) {
+    schema.customSetAttributes.forEach((attribute, validator) {
+      var result = validator(instance.data);
+      if (!result.pass) {
+        _err('Custom attribute, ${attribute}, violated ${result.message}', instance.path, instance.data);
```

Comment:
Thoughts on a better error message?

Reply:
For consistency with our other error messages, maybe something like `'${attribute} violated (${result.message})'`?

Reply:
Think it's valuable to call out that this error is from a custom attribute?

Reply:
Might be good to know as a debugging thing if you're creating your own attirbutes.